openapi.yaml

```diff
@@ -221,7 +221,7 @@ components:
           $ref: "#/components/schemas/Address"
         decimal: 
           type: string
-          pattern: '^[1-9][0-9]$'
+          pattern: '^[1-9]?\d{1}$'
```

Comment:
A better pattern may be: `^[1-9]{1,3}$` to allow for 1 to 3 digit numbers

Reply:
`^[1-9]{1,3}$` this won't include 0 in ones and tens place,  how about this - `^[1-9][0-9]{0,2}` ?

Reply:
Yep, works